OpenSim/Actuators/Millard2012AccelerationMuscle.cpp

```diff
@@ -762,7 +764,7 @@ void Millard2012AccelerationMuscle::
             case 2: //Maximum number of iterations exceeded.
             {
                 setActuation(s, 0.0);
-                setFiberLength(s,m_penMdl.getOptimalFiberLength());
+                setFiberLength(s, get_optimal_fiber_length());
```

Comment:
>What is the rationale for setting values when the solution is unknown? Seems it would be safer to throw and exception and not set arbitrary values.

I can't comment on the design of this algorithm and don't have experience using `Millard2012AccelerationMuscle`. I've created an issue (#391) to investigate.